applications/callflow/src/module/cf_branch_property.erl

```diff
@@ -0,0 +1,92 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2013-2016, 2600Hz INC
+%%% @doc
+%%%
+%%% "data":{
+%%%   "action":"branch", % the only one action supported currently
```

Comment:
future use of what ? if you're not using it in the code just remove it. please elaborate on `branch_and_update or stuff like that`